.github/workflows/trigger-tests-on-stable-packages.yml

```diff
@@ -0,0 +1,11 @@
+name: Trigger tests for stable APT packages
```

Comment:
I think this should be only started by a `workflow_dispatch` trigger. We can even do that in the `.github/workflows/test-apt-packages.yml` file itself, like:
```
on:
  workflow_call:
    inputs:
      pkg_type:
        required: true
        type: string

  workflow_dispatch:
    inputs:
      pkg_type:
        description: package-type
        type: choice
        options:
          - nightly
          - stable
```

See https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/

It could be possible to schedule the nightly run there, if we add a `DEFAULT_PKG_TYPE: nightly` env variable and use it, but I am not sure if it is the correct way to go.

Reply:
@alltilla FYI, this was the last list of triggers.
(AFAIK it is planned in the future to check a pre-release package)

```
on:
  workflow_dispatch:
  push:
    tags:
      - 'syslog-ng-*'
```

BTW I like the `choice` input method. :)

Reply:
Thanks for the clarification :)